src/app.js

```diff
@@ -1,21 +1,23 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const PORT = process.env.PORT || 3000;
+const server = http.createServer((req, res) => {
+  res.setHeader('Content-Type', 'application/json');
 
-module.exports = sum;
+  const normaluzedURL = new URL(req.url, `http://localhost:${PORT}`);
+  const query = Object.fromEntries(normaluzedURL.searchParams.entries());
+  const parts = normaluzedURL.pathname.slice(1).split('/');
```

Comment:
let's handle the situation when there are multiple values for the same query parameter. Let's add them as an array